webui/react/src/components/WorkspaceCreateModal.tsx

```
@@ -132,18 +106,16 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
         );
         return resp.resourceQuotas;
       } catch (e) {
-        if (!isNonK8RMError(e)) {
-          handleError(e, {
-            level: ErrorLevel.Error,
-            publicMessage: 'Failed to fetch kubernetes resource quotas for the workspace.',
-            silent: false,
-            type: ErrorType.Server,
-          });
-        }
+        handleError(e, {
+          level: ErrorLevel.Error,
+          publicMessage: 'Failed to fetch kubernetes resource quotas for the workspace.',
+          silent: true,
```

ditto re: silent errors